reddit/reddit-connector/reddit-connector-1.0.0/org.wso2.carbon.connector/src/test/resources/testng.xml

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="ESBTestSuite" parallel="false">
+    <listeners>
+        <listener class-name="org.wso2.carbon.automation.core.PlatformExecutionManager"/>
+        <listener class-name="org.wso2.carbon.automation.core.PlatformSuiteManager"/>
+        <listener class-name="org.wso2.carbon.automation.core.PlatformAnnotationTransferManager"/>
+        <listener class-name="org.wso2.carbon.automation.core.PlatformTestManager"/>
+        <listener class-name="org.wso2.carbon.automation.core.PlatformReportManager"/>
+    </listeners>
+
+    
+    <test name="reddit-Connector-Test" preserve-order="true" verbose="2">
+        <packages>
+            <package name="org.wso2.carbon.connector.integration.test.reddit"/>
+        </packages>
+    </test>
+
+</suite>
+

```

Comment:
Fixed in https://github.com/wso2-attic/esb-connectors/commit/c61da2ff6de0ced270ed9698ed4fbd8227a92f5b